setup.py

```diff
@@ -20,7 +20,7 @@
 with open(os.path.join(here, 'README.md')) as _file:
```

Comment:
You may have to make the change here as well?

Reply:
Well, to be honest I don't know what would be a best practice here. I know only changing it here will solve the problem, since the rest doesn't have these special characters. But maybe it is a best practice to change it everywhere.

Reply:
Looks like the change you made will not work with Python 2.7.

We can try updating the content in CHANGELOG itself.